tests/run-tests.ps1

```diff
@@ -42,5 +42,10 @@ if ($IsLocalRun) {
     $env:LOCAL_RUN = 1
 }
 
-& dotnet test --filter Category=$TestCategory -c Release --logger:trx $PSScriptRoot/Microsoft.DotNet.Framework.Docker.Tests/Microsoft.DotNet.Framework.Docker.Tests.csproj
+if ($TestCategory) {
+    & dotnet test $TestCategory -c Release --logger:trx $PSScriptRoot/Microsoft.DotNet.Framework.Docker.Tests/Microsoft.DotNet.Framework.Docker.Tests.csproj
+} else  {
+  & dotnet test -c Release --logger:trx $PSScriptRoot/Microsoft.DotNet.Framework.Docker.Tests/Microsoft.DotNet.Framework.Docker.Tests.csproj
+}
```

Comment:
Should be able to dynamically construct the parameters to `dotnet test` without duplicating them and then just have one spot that calls `dotnet test`.

Reply:
Based on Michael's additional feedback I think this can be reverted to how it was.